thread/src/components/RegisterForm.js

```diff
@@ -127,6 +128,18 @@ class RegisterForm extends Component {
     }
 }
 
+let styles = StyleSheet.create({
```

Comment:
conflicts for the good, I have moved the repeatable button markup code to the common folder